frontend/src/components/crud/TableWrapper.tsx

```diff
@@ -0,0 +1,146 @@
+/* eslint  react/jsx-props-no-spreading: 0 */ // --> OFF
+import React, { useState, useEffect } from "react";
+import BTable from "react-bootstrap/Table";
+import { useTable } from "react-table";
+import EntityAPIClient from "../../APIClients/EntityAPIClient";
+
+enum EnumField {
+  "A",
+  "B",
+  "C",
+  "D",
+}
+
+type EntityResponse = {
+  id: string | number;
+  stringField: string;
+  intField: number;
+  stringArrayField: string[];
+  enumField: EnumField;
+  boolField: boolean;
+};
+
+type EntityData = {
+  id: string | number;
+  stringField: string;
+  intField: number;
+  stringArrayField: string[];
+  enumField: EnumField;
+  boolField: string;
+};
+
+const convert = (entityReponse: EntityResponse) => {
+  return {
+    id: entityReponse.id,
+    stringField: entityReponse.stringField,
+    intField: entityReponse.intField,
+    stringArrayField: entityReponse.stringArrayField,
+    enumField: entityReponse.enumField,
+    boolField: entityReponse.boolField.toString(),
+  };
+};
+
+const GetTable = (props: any) => {
+  const { data } = props;
+  const columns = React.useMemo(
+    () => [
+      {
+        Header: "id",
+
```

Comment:
Why is there a newline here lol, did the linter do this?

Reply:
Yeah